### PR TITLE
fix(Header): Correct label color in disabled header button

### PR DESCRIPTION
**Pull request by FlyersPh9**
Disabled button in the header was not getting the correct color.  Is now fixed.

Before:
Screen Shot 2021-12-10 at 4 03 52 PM

After:
Screen Shot 2021-12-10 at 4 03 36 PM

### CODE REVIEW

File: src/header/header.scss [hunk 1]
```diff
@@ -415,7 +415,7 @@
       box-shadow: 0 $iui-xxs 0 0 rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-4));
     }
 
-    .iui-label {
+    .iui-button-label {
```
Comment (mayank99) [inline src/header/header.scss:418]: Can you also change the class below from iui-icon to iui-button-icon?
image

**Review by gretanausedaite — COMMENTED** [review]
Sorry for the bug :( I think we should add item for (or just add) tests for disabled state. This could be prevented.

File: src/header/header.scss (hunk 1 above)
Comment (FlyersPh9) [reply src/header/header.scss:418]: Done.

**Comment by FlyersPh9** [comment]
> Sorry for the bug :( I think we should add item for (or just add) tests for disabled state. This could be prevented.

No worries, it happens.  I've added a test for the disabled header buttons.

**Review by mayank99 — APPROVED** [review]

**Pull request merged**